src/components/Metrics/Metrics.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react'
+
+import { Props, State } from './Metrics.types'
+
+import './Metrics.css'
+import { SceneMetrics } from 'modules/scene/types'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
```

Comment:
Import order: External module imports should go first (after react in this case)